File: python/program/Bengkel.py
from Servis import Servis


class Bengkel:
    def __init__(self, nama_bengkel, alamat_bengkel):
        self.nama_bengkel = nama_bengkel
        self.alamat_bengkel = alamat_bengkel
        self.daftar_servis = []

    def add_servis(self, servis: Servis):
        self.daftar_servis.append(servis)

File: java/program/Bengkel.java
import java.util.ArrayList;

public class Bengkel {
    private String namaBengkel;
    private String alamatBengkel;
    private ArrayList<Servis> daftarServis;

    public Bengkel(String namaBengkel, String alamatBengkel) {
        this.namaBengkel = namaBengkel;
        this.alamatBengkel = alamatBengkel;
        this.daftarServis = new ArrayList<>();
    }

    // Getter & Setter
    public String getNamaBengkel() {
        return namaBengkel;
    }

    public void setNamaBengkel(String namaBengkel) {
        this.namaBengkel = namaBengkel;
    }

    public String getAlamatBengkel() {
        return alamatBengkel;
    }

    public void setAlamatBengkel(String alamatBengkel) {
        this.alamatBengkel = alamatBengkel;
    }

    public ArrayList<Servis> getDaftarServis() {
        return daftarServis;
    }

    public void addServis(Servis servis) {
        this.daftarServis.add(servis);
    }
}
